rfc/src/rfcs/0001-mir-linker.md

```diff
@@ -108,11 +108,12 @@ where:
  reachability analysis. No goto-program will be generated.
   This will be used to compile dependencies up to the MIR level.
    `kani-compiler` will still generate artifacts with the crate's MIR.
- - `legacy`: Keep `kani-compiler` current behavior by using
-   `rustc_monomorphizer::collect_and_partition_mono_items()` which respects the crate boundary.
-   This will generate a goto-program for each crate compiled by `kani-compiler`, and it will still have the same
-   `std` linking issues.
-   *This option will be removed as part of the `rfc` stabilization.*
+ - `tests`: Use the functions marked as tests with `#[tests]` as the starting points for the analysis.
```

Comment:
Do we also include functions marked `proof`?

Reply:
Not for the `tests` mode.